src/java.desktop/share/native/libfontmanager/freetypeScaler.c

```diff
@@ -187,6 +195,8 @@ static unsigned long ReadTTFontFileFunc(FT_Stream stream,
                                           scalerInfo->font2D,
                                           sunFontIDs.ttReadBlockMID,
                                           bBuffer, offset, numBytes);
+            // This is a callback, we are not returning immediately to Java and better report exceptions now
```

Comment:
I think the comment is un-needed .. since the only reason to call CHECK_EXCEPTION() is because of this.
Same for the other case.

Reply:
OK, comments removed.